extensions/wikia/WikiFactory/SpecialWikiFactory_body.php

```diff
@@ -712,39 +710,29 @@ private function doMultiRemoveTags( ) {
 	 *
 	 * helper function, CSS/HTML code for tab
 	 *
-	 * @author Krzysztof Krzyżaniak <eloy@wikia.com>
-	 * @access public
-	 * @static
-	 *
 	 * @param string	$tab		current tab
 	 * @param string 	$active		active tab
 	 * @param integer	$city_id	wiki id in city_list table
-	 * @param string	$tabtext	text to use that is not $tab
+	 * @param string	$tabText	text to use that is not $tab
 	 *
 	 * @return string HTML/CSS code
 	 */
-	static public function showTab( $tab, $active, $city_id, $tabtext=null ) {
+	static public function showTab( $tab, $active, $city_id, $tabText = null ) {
 
 		$title = Title::makeTitle( NS_SPECIAL, "WikiFactory/{$city_id}/{$tab}" );
 
-		if( empty($tabtext) ) {
+		if ( empty( $tabText ) ) {
 			$text = wfMsg( "wikifactory-label-{$tab}" );
-		}
-		else {
-			$text = wfMsg( "wikifactory-label-{$tabtext}" );
+		} else {
+			$text = wfMsg( "wikifactory-label-{$tabText}" );
 		}
 
-		if( $tab === $active ) {
-			#return $text;
-			$attribs = array(
-				"href" => $title->getFullUrl()
-			);
+		if ( $tab === $active ) {
+			# return $text;
+			$attribs = [ "href" => $title->getFullURL() ];
 			return Xml::element( 'a', $attribs, $text );
-		}
-		else {
-			$attribs = array(
-				"href" => $title->getFullUrl()
-			);
+		} else {
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) This branch's code block is the same as the block for the branch on line 730. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS1871)